### ISSUE TITLE
Deletion of a file does not trigger the sync action. 

### ISSUE THREAD

**Issue by Leocete** [BetaHuhn/repo-file-sync-action#68]
**🐞 Describe the bug**

Deletion of a file does not trigger the sync action. 

**📚 To Reproduce**

1. Configure repo sync action on a particular directory
2. Add some files there.
3. Delete one of the files.

**💡 Expected behavior**

A new PR opened with file deletion.

**🖼️ Screenshots**

**⚙️ Environment**
- Action version: [e.g. v1.0.0]

**📋 Additional context**

**Comment by Leocete** [comment]
A problem is here in that command:
https://github.com/BetaHuhn/repo-file-sync-action/blob/0fc61d9361931d49e2bbe3eb014cba535fc8abe1/src/git.js#L126

It somehow can't see deleted files.

**Comment by BetaHuhn** [comment]
Hey @Leocete,

Thanks for taking the time to open this issue!

`repo-file-sync-action` currently can't delete files when syncing two directories. This is not documented anywhere explicitly, which was not ideal on my part.

It only copies files which exist in the source directory to their target destination and creates a commit if there were any changes. It thus doesn't delete any files in the target directory. As you can see here:
https://github.com/BetaHuhn/repo-file-sync-action/blob/0fc61d9361931d49e2bbe3eb014cba535fc8abe1/src/index.js#L63-L68

The copying itself is done with [fs-extra](https://github.com/jprichardson/node-fs-extra/blob/master/docs/copy.md).

I currently have no fix for this, so you are welcome to propose ideas on how we can implement this in an elegant way!

**Comment by BetaHuhn** [comment]
One way to fix this would be to get all files for both the source and target and then figure out which files should be copied from the source to the target and which files should be deleted from the target.

Right now it just merges the source directory with the target.

**Comment by Leocete** [comment]
I opened a PR on that issue - https://github.com/BetaHuhn/repo-file-sync-action/pull/69

**Comment by BetaHuhn** [comment]
:tada: This issue is resolved in version [v1.9.0](https://github.com/BetaHuhn/repo-file-sync-action/releases/tag/v1.9.0) :tada:

You have to enable deleting files in the target manually with the [`deleteOrphaned`](https://github.com/BetaHuhn/repo-file-sync-action/tree/v1.9.0#delete-orphaned-files) option.

To use the latest version, use the Action with the `v1` tag:

```yml
uses: BetaHuhn/repo-file-sync-action@v1
```

Or the `latest` tag:

```yml
uses: BetaHuhn/repo-file-sync-action@latest
```
